cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml

```diff
@@ -7,26 +7,23 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: heapster-v15
+  name: heapster-v1.0.0-beta1
   namespace: kube-system
   labels:
     k8s-app: heapster
-    version: v15
```

Comment:
AFAIK [kube-addon-update.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/saltbase/salt/kube-addons/kube-addon-update.sh) uses only rc name for update purposes, so this should fine (assuming GKE is using mentioned script for updating addons).

It would be great if someone can confirm that.